ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -63,6 +63,9 @@ class TransactionsScreen extends React.Component<Props> {
           navigateToTransactionDetails={
             this.props.navigateToTransactionDetailsScreen
           }
+          noTransactionsDetailsMessage={I18n.t(
+            "wallet.noTransactionsDetailsWalletSpecific"
```

Comment:
I would call the labels after the names of the screens they're used into - i.e.

* `noTransactionsInTransactionsScreen`
* `noTransactionsInWalletHome`